changelog/README.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
 ## 9th January 2025
+* Deprecated operation extent `Documents` in [Get all customers](../operations/customers.md#get-all-customers) and [Search customers](../operations/customers.md#search-customers). Use [Get all identity documents](../operations/identitydocuments.md#get-all-identity-documents) instead.

```

Comment:
@VBuben would it be really annoying to ask you to reformat the changelog into our recommended new format? Probably yes, but I'm going to ask anyway. So, for example:
* \<API Operation\>: Deprecated blah blah"
* \<API Operation\>: Extended blah blah"

Reply:
This one was tricky... can you check it out if it looks Ok?